Makefile

```diff
@@ -42,21 +39,21 @@ test: ## Tests a previous build Docker image to see if starts
 		(echo "[test] Docker image not found, run 'make test'" && false)
 	@docker run -d --name=$(NAME_TEST_CONTAINER) \
 		-v /var/run/docker.sock:/var/run/docker.sock \
-  $(NAME_BUILD_CONTAINER) -t $(LOGENTRIES_TOKEN) -j -a host=$(NAME_TEST_CONTAINER)  > /dev/null 2>&1
+  $(NAME_BUILD_CONTAINER) -t $(LOGENTRIES_TOKEN) -r eu -a host=$(NAME_TEST_CONTAINER)  > /dev/null 2>&1
 	@echo "[test] Testing if the container stays running"
 	@echo "[test] Waiting for $(WAIT_TIME) seconds"
 	@sleep $(WAIT_TIME)
-	@docker ps | grep $(NAME_TEST_CONTAINER) | wc -l
+	@docker ps | grep -q $(NAME_TEST_CONTAINER)
 	@echo "[test] Cleaning up test container $(NAME_TEST_CONTAINER)"
 	@docker rm -f $(NAME_TEST_CONTAINER) > /dev/null 2>&1 || true
 
 tag: ## Tags local build image to make it ready for push to Docker registry
-	docker tag "$(shell docker images -q ${NAME_BUILD_CONTAINER})" "${DOCKER_REGISTRY_PREFIX}:${DOCKER_REGISTRY_IMAGE_TAG_PREFIX}${DOCKER_REGISTRY_IMAGE_VERSION}"
-	docker tag "$(shell docker images -q ${NAME_BUILD_CONTAINER})" "${DOCKER_REGISTRY_PREFIX}:${DOCKER_REGISTRY_IMAGE_TAG_PREFIX}latest"
+	docker tag "$(shell docker images -q ${NAME_BUILD_CONTAINER})" "${DOCKER_REGISTRY_PREFIX}:${DOCKER_REGISTRY_IMAGE_VERSION}${DOCKER_REGISTRY_IMAGE_TAG_PREFIX}"
+	docker tag "$(shell docker images -q ${NAME_BUILD_CONTAINER})" "${DOCKER_REGISTRY_PREFIX}:latest${DOCKER_REGISTRY_IMAGE_TAG_PREFIX}"
 
 push: ## Push local versioned and latest images to the Docker registry
-	docker push "$(DOCKER_REGISTRY_PREFIX):$(DOCKER_REGISTRY_IMAGE_TAG_PREFIX)$(DOCKER_REGISTRY_IMAGE_VERSION)"
-	docker push "$(DOCKER_REGISTRY_PREFIX):$(DOCKER_REGISTRY_IMAGE_TAG_PREFIX)latest"
+	docker push "${DOCKER_REGISTRY_PREFIX}:${DOCKER_REGISTRY_IMAGE_VERSION}${DOCKER_REGISTRY_IMAGE_TAG_PREFIX}"
```

Comment:
Could changing `$()` to `${}` cause any shell compatibility problems?

Reply:
You mean in terms of bash/shell versions?

Reply:
`${}` explicitly performs parameter expansion, `$()` is typically used for command substitution ([gnu docs](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Shell-Parameter-Expansion))

Would be almost certain this wouldn't cause compatibility problems